data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -300,7 +300,7 @@
 
         [message]
             role=Supporter
-            message= _ "Great idea, Tallin. It's so dark I probably couldn’t even fight a bat down here."
+            message= _ "Great idea, Tallin. It’s so dark I probably couldn’t even fight a bat down here."
```

Comment:
`#po: sarcasm about the idea of entering the mines where this scenario takes place`

Reply:
Outside the scope of the PR, but I've added this for you.